CalculatorTests/CalculatorTests/CalculatorTests.cs

```diff
@@ -0,0 +1,233 @@
+            driver.Url = "http://127.0.0.1:8080/Login";
+            loginfld = driver.FindElement(By.Id("login"));
+            passwordfld = driver.FindElement(By.Id("password"));
+            loginBtn = driver.FindElement(By.Id("loginBtn"));
+            loginfld.SendKeys("test");
+            passwordfld.SendKeys("newyork1");
+            loginBtn.Click();
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            driver.Quit();
+        }
+
+        [TestCase("", "", "")]
+        [TestCase("0", "0", "0")]
+        [TestCase("0", "10", "365")]
+        [TestCase("0", "0", "365")]
+        public void EmptyOrZeroDepositTest(string depositAmount, string roi, string investmentTerm)
+        {
+            // arrange
+            IWebElement amount = driver.FindElement(By.Id("amount"));
+            IWebElement percent = driver.FindElement(By.Id("percent"));
+            IWebElement term = driver.FindElement(By.Id("term"));
+            IWebElement d365 = driver.FindElement(By.Id("d365"));
+
+        // act
+        amount.SendKeys(depositAmount);
+            percent.SendKeys(roi);
+            term.SendKeys(investmentTerm);
+            d365.Click();
+
+            // assert
+            string expectedIncome = "0.00";
+            string expectedInterest = "0.00";
+            string actualIncome = driver.FindElement(By.Id("income")).GetAttribute("value");
+            string actualInterest = driver.FindElement(By.Id("interest")).GetAttribute("value");
+            
+            Assert.AreEqual(expectedIncome, actualIncome);
+            Assert.AreEqual(expectedInterest, actualInterest);
+        }
+
+
+        [TestCase("1000", "0", "0")]
+        [TestCase("1000", "10", "0")]
+        [TestCase("1000", "0", "365")]
+        public void NoInterestDepositTest(string depositAmount, string roi, string investmentTerm)
+        {
+            // arrange
+            IWebElement amount = driver.FindElement(By.Id("amount"));
+            IWebElement percent = driver.FindElement(By.Id("percent"));
+            IWebElement term = driver.FindElement(By.Id("term"));
+            IWebElement d365 = driver.FindElement(By.Id("d365"));
+
+            // act
+            amount.SendKeys(depositAmount);
+            percent.SendKeys(roi);
+            term.SendKeys(investmentTerm);
+            d365.Click();
+
+            // assert
+            string expectedIncome = "1000.00";
+            string expectedInterest = "0.00";
+            string actualIncome = driver.FindElement(By.Id("income")).GetAttribute("value");
+            string actualInterest = driver.FindElement(By.Id("interest")).GetAttribute("value");
+
+            Assert.AreEqual(expectedIncome, actualIncome);
+            Assert.AreEqual(expectedInterest, actualInterest);
+        }
+
+
+        [TestCase("1000", "10", "365")]
+        public void PositiveDepositTest_365(string depositAmount, string roi, string investmentTerm)
```

Comment:
You can combine this test with the next one:

```suggestion
        [TestCase("1000", "10", "365")]
        [TestCase("1000", "10", "360")]
        public void PositiveDepositTest_365(string depositAmount, string roi, string investmentTerm)
```

How? See the next comment.